modulemd/v2/include/modulemd-2.0/private/modulemd-translation-entry-private.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is part of libmodulemd
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Fedora-License-Identifier: MIT
+ * SPDX-2.0-License-Identifier: MIT
+ * SPDX-3.0-License-Identifier: MIT
+ *
+ * This program is free software.
+ * For more information on the license, see COPYING.
+ * For more information on free software, see <https://www.gnu.org/philosophy/free-sw.en.html>.
+ */
+
+#pragma once
+
+#include <glib-object.h>
+#include <yaml.h>
+
+#include "modulemd-translation-entry.h"
+
+/**
+ * SECTION: modulemd-translation-entry-private
+ * @title: Modulemd.TranslationEntry (Private)
+ * @stability: Private
+ * @short_description: #Modulemd.TranslationEntry methods that should be used only
+ * by internal consumers.
+ */
+
+/**
+ * modulemd_translation_entry_parse_yaml:
+ * @parser: (inout): A libyaml parser object positioned at the beginning of a
+ * Translation Entry entry in the YAML document.
+ * @error: (out): A #GError that will return the reason for a parsing or
+ * validation error.
+ *
+ * Returns: (transfer full): A newly-allocated #ModulemdTranslationEntry object
+ * read from the YAML. NULL if a parse or validation error occurred and sets
+ * @error appropriately.
+ *
+ * Since: 2.0
+ */
+ModulemdTranslationEntry *
+modulemd_translation_entry_parse_yaml (yaml_parser_t *parser, GError **error);
+
+static void *
+modulemd_translation_entry_parse_yaml_profiles (yaml_parser_t *parser,
```

Comment:
This is an internal implementation detail of the parser for the TranslationEntry. Do you see any reason why another caller in libmodulemd would need to access this directly?

I think this should probably be dropped from the header and just implemented as a static function in the C file.